apps/app/src/configs/fulfillment.neutron.ts

```diff
@@ -63,5 +59,25 @@ export const fulfillmentSettings: FulfillmentSettings = {
     ],
     sendToPatient: false,
     sendToPatientUsers: []
+  },
+  // Summer Health
+  [process.env.REACT_APP_SUMMER_HEALTH_ORG_ID as string]: {
+    sendOrder: true,
+    pickUp: true,
+    mailOrder: false,
+    mailOrderProviders: [],
+    sendToPatient: true,
+    sendToPatientUsers: []
+  },
+  // MisterRx
+  [process.env.REACT_APP_MISTERRX_ORG_ID as string]: {
+    sendOrder: true,
+    pickUp: false,
+    mailOrder: true,
+    mailOrderProviders: [
+      ...(process.env.REACT_APP_CUREXA_ORG_ID ? [process.env.REACT_APP_CUREXA_ORG_ID] : [])
+    ],
+    sendToPatient: false,
+    sendToPatientUsers: []
```

Comment:
might need to rebase or merge boson on? I think these changes are unrelated to yours